release/models/gnsi/openconfig-gnsi-acctz.yang

```diff
@@ -30,7 +27,14 @@ module openconfig-gnsi-acctz {
     "This module provides counters of gNSI accountZ requests and responses and
     the quantity of data transferred.";
 
-  oc-ext:openconfig-version "0.2.0";
+  oc-ext:openconfig-version "1.0.0";
```

Comment:
Please set the version to 0.3.0

This is because we still consider this model to be new/experimental and undergoing breaking changes

Reply:
That makes sense, I've made this change